### PR TITLE
small fix for graph js alert

### PR DESCRIPTION
**Pull request by shachar058**
Currently when there is only 1 vital form the alert states that there is no data.
I changed it to 'not enough data'.
Additionally if no item name was passed added 'This item' instead of leaving the name blank (otherwise the beginning of the sentence is just blank.

### CODE REVIEW

File: interface/patient_file/encounter/trend_form.php [hunk 1]
```diff
@@ -98,8 +98,11 @@ function show_graph(table_graph, name_graph, title_graph)
         },
         error: function() {
             // hide the chart div
-          $('#chart').hide();  
-          alert(title_graph + " " + "<?php echo xlt('has no data to graph');?>" + ".\n" +"<?php echo xlt('Please select an item that has data');?>" + ".");
+          $('#chart').hide();
+          if(!title_graph){
+              title_graph = 'This item';
```
Comment (bradymiller) [inline interface/patient_file/encounter/trend_form.php]: To make this translatable, best to place the entire alert here:
``````
alert("<?php echo xlt('This item does not have enough data to graph');?>" + ".\n" +"<?php echo xlt('Please select an item that has more data');?>" + ".");
``````
And then place the below alert() behind and `else` clause

Comment (shachar058) [reply interface/patient_file/encounter/trend_form.php]: You're right, my bad.

**Comment by shachar058** [comment]
@bradymiller 
Fixed it, waiting for your approval.
thanks

**Comment by bradymiller** [comment]
hi @shachar058 , Looks good and will bring into codebase. Thanks for the improvements! -brady

**Pull request merged**